indigo_app/templates/indigo_api/template_parts/provisions_tree_view.html

```diff
@@ -0,0 +1,18 @@
+{% for prov in provisions %}
+  <li class="list-group-item">
+    <div class="d-flex align-items-center">
+      <input id="{{ prov.id }}"
```

Comment:
need to figure out how to create unique ids otherwise we have duplicate ids

Reply:
can you prepend the parent id? that should ensure uniqueness (I think)